neural_insights/README.md

```diff
@@ -101,9 +103,15 @@ q_model = fit(
     model="./mobilenet_v1_1.0_224_frozen.pb",
     conf=PostTrainingQuantConfig(diagnosis=True),
     calib_dataloader=dataloader,
-    eval_dataloader=dataloader)
+    eval_dataloader=dataloader,
+    eval_metric=top1
+)
 ```
 
+When the quantization is finished, the workload should appear on the Neural Insights page.

```

Comment:
When the quantization is started? The graph should appear in the very beginning.